routes/aliemcards/messages.ts

```diff
@@ -0,0 +1,43 @@
+import * as express from 'express';
+import { requireAuthentication } from '../../helpers/authentication';
+const router = express.Router();
+const CHANNEL_ID = '**********'; // #aliemcards
+
+export default function route(bot: Botkit.Bot): express.Router {
+
+    router.post('/contact-form', requireAuthentication, (req, res) => {
+        const { name, email, message } = JSON.parse(req.body.data.replace(/\r\n/g, '\\n'));
```

Comment:
The replace line is necessary from wordpress because the message will be coming from a `textarea` element. You might not need this if your message will be coming from something more robust.